CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## [Unreleased]
 
+- Allow `.yml` extension for YAML files. ([#161](https://github.com/torchbox/django-pattern-library/issues/161))
```

Comment:
This should be under a heading as per https://keepachangelog.com/en/1.0.0/ 

I think "Changed" is the most appropriate here.